index.html

```diff
@@ -179,8 +207,13 @@
             }
 
             const editButton = document.createElement("button");
-            editButton.textContent = "Edit";
             editButton.classList.add("btn", "waves-effect", "waves-light", "edit-button");
+            card.appendChild(editButton);
+
+            const editButtonIcon = document.createElement("i");
+            editButtonIcon.classList.add("material-icons");
+            editButtonIcon.textContent = "edit";
+            editButton.appendChild(editButtonIcon);
```

Comment:
Replace the existing createCard() function in your code with this updated version. This implementation will use a Material Icon for the "Edit" button in all cards, just like the title edit button.